### PR TITLE
update build to set the CFNDSL_SPEC_VERSION

### PR DESCRIPTION
**Pull request by aaronwalker**

### CODE REVIEW

**Review by Guslington — APPROVED** [review]
lgtm

File: Makefile [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 RUN_RUBY_CMD=docker-compose run --rm -v $$PWD:/src -w /src ruby
 CFHL_DOCKER_TAG ?= $(shell cat lib/cfhighlander.version.rb  | grep VERSION | cut -d '=' -f 2 | sed 's/\.freeze//' | sed 's/"//g')
+CFNDSL_SPEC_VERSION=34.0.0
```
Comment (toshke) [inline Makefile:5]: perhaps update to `CFNDSL_SPEC_VERSION ?=` so it can be passed as env variable.

**Review by toshke — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (aaronwalker) [reply Makefile]: good idea.. updated PR

**Review by toshke — APPROVED** [review]
Lgtm

**Pull request merged**